benchmarks/Benchmarks.Trace/TracerBenchmark.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Jobs;
+using Datadog.Trace;
+using Datadog.Trace.BenchmarkDotNet;
+using Datadog.Trace.ExtensionMethods;
+
+namespace Benchmarks.Trace
+{
+    /// <summary>
+    /// Tracer benchmarks
+    /// </summary>
+    [DatadogExporter]
+    [MemoryDiagnoser]
+    [SimpleJob(RuntimeMoniker.Net472)]
+    [SimpleJob(RuntimeMoniker.NetCoreApp31)]
+    public class TracerBenchmark
+    {
+        static Action<Tracer> _runShutdownTasks;
+
+        static TracerBenchmark()
+        {
+            _runShutdownTasks = (Action<Tracer>)typeof(Tracer)
```

Comment:
Maybe we should just expose the agents with InternalsVisibleTo, so that it doesn't break silently if we rename the method

Reply:
I've removed the `TracerBenchmark` because needs works to do a proper benchmark of the tracer, and the goal for the benchmark tests in this PR is to produce some data to the exporter.